ui/apps/platform/src/Components/ReportJobsHelpAction.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Popover, TabAction } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import { systemConfigPath } from 'routePaths';
+import PopoverBodyContent from './PopoverBodyContent';
+
+type ReportJobsHelpActionProps = {
+    reportType: 'Vulnerability' | 'Scan schedule';
+};
+
+function ReportJobsHelpAction({ reportType }: ReportJobsHelpActionProps) {
+    return (
+        <Popover
+            aria-label="All report jobs help text"
+            bodyContent={
+                <PopoverBodyContent
+                    headerContent="All report jobs"
+                    bodyContent={
+                        <>
+                            This function displays the requested jobs from different users and
+                            includes their statuses accordingly. While the function provides the
+                            ability to monitor and audit your active and past requested jobs, we
+                            suggest configuring the{' '}
+                            <Link to={systemConfigPath}>{reportType} report retention limit</Link>{' '}
```

Comment:
Because **System Configuration** page requires **Administration** resource:
* link need conditional rendering whether to include it
* therefore, grammar probably needs rearrangement to include last instead of in middle of sentence

Probably also a decision whether to open the page in a new tab?

If so, it needs `target="_blank" rel="noopener noreferrer">` for `Link-target-rel` lint rule in #12558

Reply:
Also, you have a good point about the page access. We can display two different texts based on the user's access.

Reply:
I also decided to clean up the wording to make it clearer. Looks much better:

With **Read/Write Access**
<img width="1440" alt="Screenshot 2024-09-04 at 2 21 10 PM" src="https://github.com/user-attachments/assets/09b9d975-a1d2-4463-83c6-2c30db060939">

With **Read Access** or **No Access**
<img width="1440" alt="Screenshot 2024-09-04 at 2 18 12 PM" src="https://github.com/user-attachments/assets/b1ea169b-6818-4f96-bb5d-8c14af445213">